openapi3.yaml

```diff
@@ -290,6 +290,7 @@ components:
           type: string
     configName:
       type: string
+      pattern: '^[a-z0-9\-]+$'
```

Comment:
This pattern allows multiple `-` in a row, don't we want to limit this to one in a row and not at the beginning and the end?
Something like this:
`^([a-z0-9]+(-[a-z0-9]+)*)$`

Reply:
fixed

Reply:
Did you make sure my regex is correct?